src/account/ShakeForSupport.test.tsx

```diff
@@ -22,24 +20,4 @@ describe('ShakeForSupport', () => {
     )
     expect(tree).toMatchSnapshot()
   })
-
-  it('renders correctly when App is active', async () => {
-    const tree = render(
-      <Provider store={createStore(AppState.Active)}>
-        <ShakeForSupport />
-      </Provider>
-    )
-    expect(tree.queryByTestId('ContactSupportFromShake')).toBeTruthy()
-    expect(tree).toMatchSnapshot()
-  })
-
-  it('navigates to support screen after button press', async () => {
-    const { getByTestId } = render(
-      <Provider store={createStore(AppState.Active)}>
-        <ShakeForSupport />
-      </Provider>
-    )
-    fireEvent.press(getByTestId('ContactSupportFromShake'))
-    expect(navigate).toHaveBeenCalledWith(Screens.SupportContact)
-  })
```

Comment:
Why are we removing these?

Reply:
oh sorry, should have explained.

it relates to the key change where <Modal visible={false} /> components are no longer rendered in the test dom. notice how in this unit test, we never bother to actually _launch_ the bottom sheet - we make all these assertions and actions but the whole time, the Modal has `visible={false}`.

the launching of the sheet is done by shake action. i couldn't figure out how to do this with this testing lib. we also cover this exact test in the e2e tests so i thought it was okay to okay to remove these

Reply:
Oh ok, thanks, if it's cover by the e2e test it's all good then 👍